new_cour.py

```diff
@@ -195,24 +197,25 @@ def check(hash_list, course_id, direc):
     if DEBUG:
         print("Course : " + str(course_id))
 
-    pynotify.init("11")
     ret = hash_foo('resources.php', course_id)
     if ret != hash_list[1] and ret != -1:
         test('http://courses.iiit.ac.in/EdgeNet/resources.php?select=%s' % (course_id), direc + hash_list[0] + '/resources/')
         hash_list[1] = ret
-        pynotification = pynotify.Notification(hash_list[0], "Resources Updated!  http://courses.iiit.ac.in/EdgeNet/resources.php?select=%s" % (course_id), os.path.join(RUNNING_DIRECTORY, "iiith_logo.gif"))
-        pynotification.show()
+        pymsgbox.native.alert("Resource Updated! @ http://courses.iiit.ac.in/EdgeNet/resources.php?select=%s" % (course_id), 'Resource Updated!')
+        webbrowser.open("http://courses.iiit.ac.in/EdgeNet/resources.php?select=%s" % (course_id))
+
     ret = hash_foo('assignments.php', course_id)
     if ret != hash_list[2] and ret != -1:
         test('http://courses.iiit.ac.in/EdgeNet/assignments.php?select=%s' % (course_id), direc+hash_list[0]+'/assignments/')
         hash_list[2] = ret
-        pynotification = pynotify.Notification(hash_list[0], "Assignments Updated!  http://courses.iiit.ac.in/EdgeNet/assignments.php?select=%s" % (course_id), os.path.join(RUNNING_DIRECTORY, "iiith_logo.gif"))
-        pynotification.show()
+        pymsgbox.native.alert("Assignments Updated!  http://courses.iiit.ac.in/EdgeNet/assignments.php?select=%s" % (course_id), 'Assignments Updated!')
```

Comment:
I am not liking pymsgbox any better than libnotify.

Any reason you wish to switch to it?